### ISSUE TITLE
Component: Calendar Selection of date when screen reader is active is not possible 

### ISSUE THREAD

**Issue by Draganlazarev90** [primefaces/primeng#15333]
### Describe the bug

When having screen reader active you cannot select a date. The screen reader reads the dates with up/down keys but on pressing enter/space nothing happens

### Environment

17.14.1

### Angular version

17.0.5

### PrimeNG version

17.4.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0

### Steps to reproduce the behavior

1. Activate a screen reader
2. Go to a date from the calendar and try to select it with space/enter

### Expected behavior

Expected to be able to select a date from the picker when screen reader is active

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you open a new issue so we can include it in our roadmap?

Thanks a lot for your understanding!
Best Regards,
